evaluation/benchmarks/swe_bench/loc_eval/README.md

```diff
@@ -0,0 +1,81 @@
+# **SWE-Bench Evaluation with Localization Evaluation**
+
+This folder implements localization evaluation at both file and function levels to complementing the assessment of agent inference on [SWE-Bench](https://www.swebench.com/).
+
+## **1. Environment Setup**
+- Python env: [Install python environment](../../../README.md#development-environment)
+- LLM config: [Configure LLM config](../../../README.md#configure-openhands-and-your-llm)
+
+## **2. Inference & Evaluation**
+- Run inference on SWE-Bench instances:
+    - You may refer to instructions at [README.md](../README.md) for running inference on SWE-Bench
+    - General settings:
+        - Format:
+            ```bash
+            ./evaluation/benchmarks/swe_bench/scripts/run_infer.sh [model_config] [git-version] [agent] [eval_limit] [max_iter] [num_workers] [dataset] [dataset_split] [N_RUNS] [MODE] [loc-eval-save]
+            ```
+            - `model_config`: the config group name for your LLM settings, as defined in your `config.toml`.
+            - `git-version`: the git commit hash of the OpenHands version you would like to evaluate.
+            - `agent`: the name of the agent for benchmarks, defaulting to `CodeActAgent`.
+            - `eval_limit`: limits the evaluation to the first `eval_limit` instances. By default, the script evaluates the entire SWE-bench_Lite test set (300 issues). 
+                - Note: in order to use `eval_limit`, you must also set `agent`.
+            - `max_iter`: the maximum number of iterations for the agent to run. By default, it is set to 100.
+            - `num_workers`: the number of parallel workers to run the evaluation. By default, it is set to 1.
+            - `dataset`: SWE-Bench dataset name. Please use huggingface format to specifies which dataset to evaluate on.
+            - `dataset_split`: SWE-Bench dataset split.
+            - `loc-eval-save`: directory to save outputs for localization
```

Comment:
Why do you need an additional `loc-eval-save`? You should be able to put the result into `EvalOutput` in `run_infer.py` - so we don't need an additional dir, rather, everything would be saved into the `output.jsonl` together with instance ID